css/properties/alignment-baseline.json

```
@@ -27,7 +27,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

It was already present in 4.0.5, see: https://github.com/caugner/css-properties/blame/safari/properties.css

```suggestion
              "version_added": "≤4"
```